Dockerfile

```diff
@@ -68,6 +68,7 @@ ARG HUMHUB_VERSION
 RUN apk add --no-cache \
     curl \
     ca-certificates \
+    imagemagick \
```

Comment:
To be honest, should give the developer the option to install with `imagemagick` or with `graphicsmagick`, in most cases using `graphicsmagick` would be much better on the processing end and much faster and can deal with large images, just my thoughts :thinking: